elements/rhelement/README.md

```diff
@@ -8,24 +8,27 @@ Here is an example of how RHElement should be extended
 ```
```

Comment:
mention generator and `npm run new` here, to be clear that's the correct way to create an element